### PR TITLE
Perform checking before render

### PR DESCRIPTION
**Pull request by qingweibinary**

### CODE REVIEW

**Comment by qingweibinary** [comment]
Why clear localstorage fix the problem?
Because after clear, workspace will not try to render right panel, which will be broken if trying to render on load before data is available ~~

File: src/asset-details/AssetDetailsCard.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ export default class AssetDetailsCard extends Component {
 				}
 				<br />
 				<h5><M m="Durations" /></h5>
-				<AssetDetailsDurations durations={durations} />
+				{durations && <AssetDetailsDurations durations={durations} />}
```
Comment (borisyankov) [inline src/asset-details/AssetDetailsCard.js]: Should the AssetDetailsDurations handle this?
In render if !durations return null

Comment (borisyankov) [inline src/asset-details/AssetDetailsCard.js]: Hint: the question is rethorical :dancer:

Comment (qingweibinary) [inline src/asset-details/AssetDetailsCard.js]: I think both would work, would you explain your reasoning and maybe we can generalize it as a rule so we have less decision to make =) ?

**Pull request closed without merging**